MaxMind.MinFraud/Request/CreditCard.cs

```diff
@@ -73,6 +73,31 @@ public sealed class CreditCard
             Was3dSecureSuccessful = was3dSecureSuccessful;
         }
 
+        /// <summary>
+        /// Legacy constructor so that binary compatibility isn't broken.
+        /// </summary>
+        public CreditCard(
+            string? issuerIdNumber,
+            string? last4Digits,
+            string? bankName,
+            string? bankPhoneCountryCode,
+            string? bankPhoneNumber,
+            char? avsResult,
+            char? cvvResult,
+            string? token
+        )
+        {
```

Comment:
This looks fine, but I would go ahead and call the new constructor rather than initializing everything here. The reason is that it will ensure we don't forget to update this one if we add logic or something to the primary constructor, e.g.:

```
        public CreditCard(
            string? issuerIdNumber,
            string? last4Digits,
            string? bankName,
            string? bankPhoneCountryCode,
            string? bankPhoneNumber,
            char? avsResult,
            char? cvvResult,
            string? token
        ) : this(issuerIdNumber, last4Digits, bankName, bankPhoneCountryCode, bankPhoneNumber, avsResult, cvvResult, token, null)
        {
        }
```

Reply:
Makes sense!

Reply:
Maybe also mark this as obsolete given that we will remove it in the next major release, e.g.:

```
[Obsolete("Legacy constructor for backwards compatibility")]
```